### ISSUE TITLE
Job 252337 stuck in an odd state

### ISSUE THREAD

**Issue by Hussell** [MotusDev/Motus-TO-DO#418]
Job 252337 has been waiting for 2 days now (since 2018-12-12), and the subjob completion/wait states have a strange pattern. (Not completely sure this isn't just something I haven't seen before.)

https://sgdata.motus.org/status?jobID=252337

**Comment by jbrzusto** [comment]
The log for subjob 252385 (scroll down from [here](https://sgdata.motus.org/status?jobID=252337)) shows 

jobID|type|mtime|status|params
---|---|---|---|---
252385 | LtFindtags | 2018-12-12 16:28:31.155Z | Waiting | **serno = null**; tsStart = Inf

which (bold) indicates the serial number wasn't parsed correctly from at least one of the submitted files.  These "weird" files ended up in `/sgm/trash/252337`, where we see this:
```bash
$ ls -l
-rw-r--r-- 1 sg sg 1098883 Nov  4 14:26 ERI_04Nov2018.DTA
-rw-r--r-- 1 sg sg  576937 Dec  2 10:11 STA_02Dec2018.DTA
$ grep -F 'S/N' *
ERI_04Nov2018.DTA:Receiver S/N:    
ERI_04Nov2018.DTA:Receiver S/N:  
...  
STA_02Dec2018.DTA:Receiver S/N:    
STA_02Dec2018.DTA:Receiver S/N: 
...
```
i.e. each line in the .DTA file where the receiver serial number is supposed to appear shows
no serial number.

I'm (really just) guessing these files were generated by a version of the Lotek client software too old for
the version of the receiver firmware.  

Regardless, the server should do something more intelligent than just silently dropping the job,
so I'm filing jbrzusto/motusServer#427.

The user and/or Lotek might find the following .DTA file snippets useful:

### ERI_04Nov2018.DTA
```
SRX-DL Information:
Environment History:
changed: 08/16/18  15:27:30

Master Firmware: Version 1.9
Slave Firmware:  Version 3.0
Receiver S/N:    

Active Configuration: 1
Site Code: 8205
Code Set: Lotek4
```
and
### STA_02Dec2018.DTA
```
SRX600 Information:
Environment History:
changed: 10/26/18  15:00:15

Master Firmware: Version 8.3
Slave Firmware:  Version 11.0
Receiver S/N:    

Active Configuration: 1
Site Code: 6347
Code Set: Lotek4
```

**Comment by Hussell** [comment]
I got an email from the user: she suspects the receiver with the bad log file is Lotek-8205. Hope that helps.

**Comment by jbrzusto** [comment]
This job is now re-running. (fixed jbrzusto/motusServer#427)
